.github/workflows/update-nextcloud-ocp.yml

```diff
@@ -0,0 +1,116 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Update nextcloud/ocp
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "5 2 * * 0"
+
+jobs:
+  update-nextcloud-ocp:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        branches: ['main', 'master', 'stable28', 'stable27', 'stable26']
```

Comment:
both master and main? Will it always fail for one of them?

Reply:
Ah true since this app has no branching we don't need any of this. I just blindly copied the workflow from the template repository without checking.